lib/kamal/commands/lock.rb

```diff
@@ -21,6 +21,10 @@ def status
       read_lock_details
   end
 
+  def ensure_locks_directory
```

Comment:
If we put the `-p` on line 8 above we don't need this extra step to create the `locks` dir here. It will get created on demand when we acquire a lock.

Reply:
No, because in this case it won't raise an error if `locks/app` exists and `Kamal::Base#raise_if_locked` won't be called.

Reply:
Ah! Yes that makes sense 👍

In that case there's no need to create the run directory and locks directories separately - let's combine creating the run and locks directories into one command - `ensure_run_and_locks_directory` and get rid of `ensure_run_directory`.

Reply:
I split this into separate methods because `ensure_run_directory` ran on all hosts, while `ensure_locks_directory` ran only on a primary host.

I combined them into a single method.